src/SourceBuild/content/repo-projects/roslyn-analyzers.proj

```diff
@@ -8,8 +8,8 @@
 
   <ItemGroup>
     <RepositoryReference Include="arcade" />
-    <RepositoryReference Include="runtime" />
-    <RepositoryReference Include="roslyn" />
```

Comment:
No roslyn dep? This is surpising...does it just target surface area of roslyn?

Reply:
Yeah, it's not based on the live version.